mk/rt.mk

```diff
@@ -384,7 +384,11 @@ COMPRT_OBJS_$(1) += emutls.o
 endif
 
 ifeq ($$(findstring msvc,$(1)),)
+
+ifeq ($$(findstring freebsd,$(1)),)
```

Comment:
Correct, the first condition checks for "not msvc" and the second checks for "not freebsd".